### DISCUSSION TITLE
Channel membership-only content

### DISCUSSION THREAD

**Discussion by SaimonWoidig** [TeamNewPipe/NewPipe#10652] [Questions]
First of all, this applies to YouTube.
Are there any plans to support premium per-channel content somehow? I don't know how it will fit into the whole anonymous browsing but if it were a way to fetch videos via my account or a token maybe.

**Comment by opusforlife2** [comment]
I'm not sure, but I think membership stuff needs DRM, which is hard for FOSS apps to deal with. See how Firefox has to work around it.

Moreover, no logins is considered a feature, not a drawback, so I don't know if there will be any enthusiasm around implementing related functionality.

But youtube-dl allows manually using cookies to do things like these, so it might be possible.
